mobile/src/classes/WordsAliveAPI.tsx

```diff
@@ -0,0 +1,18 @@
+import axios, { AxiosInstance } from 'axios';
+import { Book } from '../models/Book';
+
+// Class to encapsulate the handler for the Words Alive API
+class WordsAliveAPI {
+    client: AxiosInstance;
+
+    constructor(baseURL: string) {
+      this.client = axios.create({ baseURL: baseURL });
+    }
+
+    async getBooks(): Promise<Book[]> {
+      const res = await this.client.get('/books');
+      return res.data;
+    }
+}
+
+export { WordsAliveAPI };
```

Comment:
Can we move this to a `src/api/` dir instead of classes? Should make it easier to find and extend if we want to. Also can you document getBooks()? Lastly can you indent this to 2 spaces? No idea why the lint didn't pick that up.